store/storetest/channel_store.go

```diff
@@ -153,7 +153,7 @@ func testChannelStoreSaveDirectChannel(t *testing.T, ss store.Store, s SqlSuppli
 		t.Fatal("couldn't save direct channel", err)
 	}
 
-	members := (<-ss.Channel().GetMembers(o1.Id, 0, 100)).Data.(*model.ChannelMembers)
+	members, _ := ss.Channel().GetMembers(o1.Id, 0, 100)
```

Comment:
Not in the previous code, but would be great to add here a `require.Nil(t, err)` to verify the error instead of ignore it (and the same for the other cases in this file).